recipes/ribohmm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: ribohmm
+  version: "1.0.0"
+
+source:
+  url: https://github.com/rajanil/riboHMM/archive/57a34d5afbd329d9641c895d12d1b0cefea0735b.zip
+  sha256: b0e182e0e07ede199aafef1936e2eab21fa21c6d15ffb1e00386bc0db167b436
+
+build:
+  number: 1
+  skip: True [osx or py3k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - numpy 1.11
```

Comment:
I don't think you should pin numpy here.
Is this tool compiling against the numpy ABI?

Reply:
Problem is that it failed with newer numpy versions, so i downgraded